agir/events/components/eventPage/EventPhotosCard.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import styled from "styled-components";
+import Button from "../../../front/components/genericComponents/Button";
+import { DateTime } from "luxon";
+import Card from "../../../front/components/genericComponents/Card";
+
+const Thumbnails = styled.div`
+  display: grid;
+  grid-gap: 0.5rem;
+  grid-template-columns: repeat(auto-fill, 160px);
+  grid-auto-rows: 160px;
+  align-items: center;
+  justify-items: center;
+
+  a {
+    display: block;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+  }
+
+  img {
+    margin: 0;
+    width: 100%;
+    height: 100%;
+    transform: scale(1);
+    transition: transform 100ms ease-in-out;
+
+    &:hover,
+    &:focus {
+      transform: scale(1.01);
+      transform-origin: center center;
+    }
+  }
+`;
+
+const StyledButton = styled(Button)`
+  height: 48px;
+  font-size: 16px;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```